style/CONTRIBUTE.md

```diff
@@ -180,11 +180,12 @@ Highway shields are a key feature of the OpenStreetMap Americana style. This gui
 
 In addition, the following style guidelines are recommended:
 
-1. Use a 1px stroke size for line elements. Horizontal/vertical lines should be aligned to the pixel grid. In Inkscape, use Shift+Ctrl+D, and create a rectangular grid in the "Grids" tab and enable the "snap to grid" option.
+1. Use a 1px stroke size for line elements. Horizontal/vertical lines should be aligned to the pixel grid.
+   - In Inkscape, press `Ctrl+Shift+D`, and create a rectangular grid in the "Grids" tab. Set Spacing X and Spacing Y to `0.5`, and Major grid line every `2`. You can turn snapping to the grid on and off by pressing `%`.
```

Comment:
Since strokes should still be 1&nbsp;pixel wide, I think a 1-pixel grid remains useful.

Reply:
When snapping a 1px stroke to a grid, the handles end up on the halfway points between pixels. Major grid lines are suggested every 2 grid lines, so that they line up with whole pixels. Personally, I've found it very helpful to use a 0.5px grid for this project.
![Screenshot from 2022-05-10 20-46-55](https://user-images.githubusercontent.com/1732117/167747120-308e9ea5-2f21-49e9-bb09-3b3309b519f9.png)